scipy/linalg/flapack_pos_def_tri.pyf.src

```diff
@@ -83,3 +83,65 @@ subroutine <prefix2c>pttrs(lower, n, nrhs, d, e, b, ldb, info)
     integer intent(out) :: info
 
 end subroutine<prefix2c>pttrs
+
+subroutine <prefix2>ptsvx(fact, n, nrhs, d, e, df, ef, b, ldb, x, ldx, rcond, ferr, berr, work, info)
+    ! DPTSVX uses the factorization A = L*D*L**T to compute the solution
+    ! to a real system of linear equations A*X = B, where A is an N-by-N
+    ! symmetric positive definite tridiagonal matrix and X and B are
+    !N-by-NRHS matrices.
+    !
+    ! Error bounds on the solution and a condition estimate are also
+    ! provided.
+    callstatement (*f2py_func)(fact, &n, &nrhs, d, e, df, ef, b, &ldb, x, &ldx, &rcond, ferr, berr, work, &info)
+    callprotoargument char*, int*, int*, <ctype2>*, <ctype2>*, <ctype2>*, <ctype2>*, <ctype2>*, int*, <ctype2>*, int*,  <ctype2>*, <ctype2>*, <ctype2>*, <ctype2>*, int*
+
+    character optional, intent(in) :: fact = 'N'
+    integer intent(hide), depend(d) :: n = max(1, len(d))
```

Comment:
both `n` and `nhrs` are allowed to be zeros. So they don't need default values. Only `e`, `ef` dimension needs guards `dimension(MAX(0, n-1))`

Reply:
OK